pkg/release/push.go

```diff
@@ -485,3 +487,39 @@ func (p *PushBuild) CopyStagedFromGCS(stagedBucket, version, buildVersion string
 
 	return nil
 }
+
+// StageLocalSourceTree creates a src.tar.gz from the Kubernetes sources and
+// uploads it to GCS.
+func (p *PushBuild) StageLocalSourceTree(buildVersion string) error {
```

Comment:
This can be refactored now that https://github.com/kubernetes/release/pull/1664 has merged.

Reply:
Yep, rebased on top of the change. Still have to give this a manual test.